similarity/config/configurations.properties

```diff
@@ -0,0 +1,20 @@
+wordMetric	paragram
```

Comment:
Actually I'm suggesting keeping both. SimConfigurator class has the default configuration while configurations.properties file make user easily change configuration. Or if you want to drop either one. I'm fine with it.